packages/websocket-async/websocket-async.2.13/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+name: "websocket-async"
+version: "2.13"
+authors: "Vincent Bernardoff <vb@luminar.eu.org>"
+maintainer: "Vincent Bernardoff <vb@luminar.eu.org>"
+homepage: "https://github.com/vbmithr/ocaml-websocket"
+bug-reports: "https://github.com/vbmithr/ocaml-websocket/issues"
+dev-repo: "git+https://github.com/vbmithr/ocaml-websocket"
+doc: "https://vbmithr.github.io/ocaml-websocket/doc"
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+build: [ "dune" "build" "-j" jobs "-p" name "@install" ]
+depends: [
+  "dune" {build & >= "1.1.1"}
+  "websocket" {= "2.12"}
```

Comment:
this doesn't seem to be the right version number

Reply:
you can now use `{=version}` to make the constraint track the current package version